do/droplets.go

```diff
@@ -133,9 +136,12 @@ func (i *instances) CurrentNodeName(hostname string) (types.NodeName, error) {
 
 // dropletById returns the godo Droplet type corresponding to the provided id
 func (i *instances) dropletById(ctx context.Context, id string) (*godo.Droplet, error) {
+
+	glog.Infof("dropletById %v", id)
+
 	intId, err := strconv.Atoi(id)
 	if err != nil {
-		return nil, fmt.Errorf("error converting droplet id to string: %v", err)
+		return nil, fmt.Errorf("error converting droplet id to string: %v %s", err, id)
```

Comment:
Might not be needed.
`Atoi` error already includes the string being converted at the error message.
https://play.golang.org/p/QZWIsCUtXc

Reply:
agreed